dvc/commands/experiments/exec_run.py

```diff
@@ -36,4 +37,13 @@ def add_parser(experiments_subparsers, parent_parser):
         help="Path to executor info file",
         default=None,
     )
+    exec_run_parser.add_argument(
+        "--copy-paths",
+        action="append",
+        default=[],
+        help=(
+            "List of paths to copy inside the temp directory."
+            " Only used if `--temp` or `--run-all` is specified."
```

Comment:
```suggestion
            "List of untracked paths to copy inside the temp directory."
            " Only used if `--temp` or `--run-all` is specified."
```

I think in the docs we should also discourage this/warn that it may break reproducibility of the experiment.

WDYT?

Reply:
Should we also say ignored? 

> I think in the docs we should also discourage this/warn that it may break reproducibility of the experiment.

Makes sense

Reply:
> Should we also say ignored?

Like `List of untracked or ignored paths`? LGTM.

Reply:
Updated

Reply:
Just checking on this and then I'm fine to approve. Thanks @daavoo!

Reply:
Oh. I missed it. 

`exec_run` is not really exposed to the public (is just used by the celery queue internally) so the help will not be used. 

Anyhow, updated the message also here for consistency